diesel/src/sql_types/mod.rs

```diff
@@ -57,7 +57,7 @@ pub struct Bool;
 /// [i8]: https://doc.rust-lang.org/nightly/std/primitive.i8.html
 #[derive(Debug, Clone, Copy, Default, QueryId, SqlType)]
 #[mysql_type = "Tiny"]
-pub struct Tinyint;
```

Comment:
We still need to keep this name around. We can't just make a major breaking change like this